### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by seb-mtl**
Bump dependencies

### CODE REVIEW

File: src/io_context.ts [hunk 1]
```diff
@@ -278,7 +276,7 @@ export function whichFilesInDirAreOpen(dirpath: string): Promise<Map<string, Fil
 
 }
 
-function getFilesystem(drive: any, mountpoint: string) {
+function getFilesystem(drive: any, mountpoint: string): Promise<FILESYSTEM> {
```
Comment (daniellanner) [inline src/io_context.ts:279]: Why is this a promise?

File: src/io_context.ts [hunk 2]
```diff
@@ -695,7 +693,8 @@ export class IoContext {
           return checkReadAccess(absPaths, relPaths);
         })
         .then(() => {
-        })
+          /* */
```
Comment (daniellanner) [inline src/io_context.ts:696]: Can the empty `then` be removed?

**Review by daniellanner — APPROVED** [review]

File: src/io_context.ts (hunk 1 above)
Comment (seb-mtl) [reply src/io_context.ts:279]: Because the function uses `exec`

File: src/io_context.ts (hunk 2 above)
Comment (seb-mtl) [reply src/io_context.ts:696]: Actually yes

Comment (seb-mtl) [reply src/io_context.ts]: Good catch!

**Review by daniellanner — APPROVED** [review]

**Pull request merged**